### ISSUE TITLE
[Request] Netherite level pickaxe is needed!

### ISSUE THREAD

**Issue by NoZoul** [lehjr/MachineMusePowersuits#105]
Describe the solution you'd like:
I would like to propose the addition of a "Netherite" mining drill attachment for the power tool. This attachment would enhance the efficiency and durability of the tool when mining in challenging environments such as the Nether, and is a necessity when playing with other mods, which a lot of blocks require Netherite levels of mining.

Describe alternatives you've considered:
One alternative could be introducing a separate Netherite-powered tool, but integrating it as an attachment to the existing power tool would provide players with a more streamlined and flexible gameplay experience.

Additional context:
As Netherite has become a significant material in the game, especially for its durability and resilience, having a specialized attachment for the power tool would not only add depth to gameplay but also align with the evolving mechanics of Minecraft.

**Comment by lehjr** [comment]
This will probably be something that happens in the 1.20.4 builds along with other changes that allow for better progression.

**Comment by NoZoul** [comment]
> This will probably be something that happens in the 1.20.4 builds along with other changes that allow for better progression.

Yes probably. It's just that, it would be fantastic if it could be added to existing versions like 1.19.2 etc.

**Comment by lehjr** [comment]
I generally try to make it a point not to make changes to established versions that change the mechanics of the mod.
